lib/view_model/wallet_creation_vm.dart

```diff
@@ -115,11 +117,23 @@ abstract class WalletCreationVMBase with Store {
   }
 
   DerivationInfo? getDefaultDerivation() {
-    switch (this.type) {
+    final useBip39 = seedSettingsViewModel.bitcoinSeedType.type == DerivationType.bip39;
+    switch (type) {
       case WalletType.nano:
         return DerivationInfo(derivationType: DerivationType.nano);
       case WalletType.bitcoin:
+        if (useBip39) {
+          return bitcoin!
+              .getElectrumDerivations()[DerivationType.bip39]!
+              .firstWhere((element) => element.description == "Standard BIP84 native segwit");
+        }
+        return bitcoin!.getElectrumDerivations()[DerivationType.electrum]!.first;
       case WalletType.litecoin:
+        if (useBip39) {
+          return bitcoin!
+              .getElectrumDerivations()[DerivationType.bip39]!
+              .firstWhere((element) => element.description == "Default Litecoin");
+        }
```

Comment:
just hard code the derivation path here better than comparing with a description string